main.js

```diff
@@ -0,0 +1,30 @@
+let customer = {
+    name: '',
+    location: '',
+    phone: '',
+    picture: '',
+    dob: ''
+}
+
+let targetEl = document.querySelector('.customer')
+
+// create a ul 
+let list = document.createElement('ul')
+
+// append the ul to the dom
+targetEl.appendChild(list)
+for (let customer of customers){
+
+// create a li
+let listItem = document.createElement('li')
+
+//  create a p element
+let itemNameEl = document.createElement('p')
+itemNameEl.innerText = customer
```

Comment:
@kimqi1 This is great progress!! This line is causing you to get that `[object Object]` business on you page. What is the value of `customer` here? Console log this to see it. You will need to extract the data from the customer object here using the key that corresponds to the value you want. So, if you want the customer name, you will need to say something liek what we saw Phil demo this morning: `customer.name.first` will give you the first name, then `customer.name.last` will give you the last name, and then you can concatenate those strings. 

You are very close to success here! You're creating the elements and adding them to the DOM successfully.